articlewidget.hh

```diff
@@ -429,36 +446,35 @@ class ResourceToSaveHandler: public QObject
 {
   Q_OBJECT
 
-public:
-  explicit ResourceToSaveHandler( ArticleView * view, QString const & fileName );
+ public:
+  explicit ResourceToSaveHandler( ArticleWidget * view, QString const & fileName );
   void addRequest( sptr< Dictionary::DataRequest > req );
-  bool isEmpty()
-  { return downloadRequests.empty(); }
+  bool isEmpty() { return downloadRequests.empty(); }
 
-signals:
+ signals:
   void done();
   void statusBarMessage( QString const & message, int timeout = 0, QPixmap const & pixmap = QPixmap() );
 
-public slots:
+ public slots:
   void downloadFinished();
 
-private:
+ private:
   std::list< sptr< Dictionary::DataRequest > > downloadRequests;
```

Comment:
seems the .clang-format file has configured wrong .

do not indent seems better.

Reply:
I fixed clang format, the problem is gone